client/src/components/settings/toggle-radio-setting.tsx

```diff
@@ -41,26 +41,43 @@ export default function ToggleRadioSetting({
           {explain ? <p id={`desc${flagName}`}>{explain}</p> : null}
         </div>
         <div className='toggle-radio-group'>
-          <label htmlFor={firstRadioId} data-checked={flag}>
+          <label
+            htmlFor={firstRadioId}
+            className='toggle-radio-label'
+            data-checked={flag}
+          >
             <input
               id={firstRadioId}
               type='radio'
+              className='hide-default-radio'
```

Comment:
Do we need this new class (`hide-default-radio`)? Can we just use `.toggle-radio-group input` instead?

Reply:
Yes, we can! It works fine.